src/Symfony/Component/Yaml/Inline.php

```diff
@@ -562,6 +564,15 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                         }
 
                         return;
+                    case 0 === strpos($scalar, '!php/const:'):
+                        if (self::$constantSupport) {
+                            return @constant(substr($scalar, 11));
```

Comment:
Is there a way to distinguish `null` when assigned to a constant from when it's not defined?